### ISSUE TITLE
Levoscoliosis

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4837]
**Preferred term label:**
Levoscoliosis

**Definition (free text, please give PubMed ID)**
A kind of scoliosis where your spine twists and curves toward the left side of your body in a C shape. The curve usually happens in the lumbar (lower) portion of the back, but it can happen in the thoracic (middle) back as well.
https://www.healthline.com/health/levoscoliosis (I couldn't find an article in PubMed - I wonder if this term is necessary, or is scoliosis sufficient?)

**Parent term (use hpo.jax.org/app)**
Scolisosis

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

For Kids First, the description says "levoscoliosis"

**Comment by pnrobinson** [comment]
"levo" and "dextro" refer to the direction of the curve in scoliosis. I do not think we want to have separate terms for them since to my knowledge there is no significant difference in terms of complications or treatment, and we would have to have "leve/dextro" terms for all of the subterms of scoliosis. I would advise annotating with just "Scoliosis". @nicolevasilevsky
